plugins/builds/update.js

```diff
@@ -7,6 +7,29 @@ const schema = require('screwdriver-data-schema');
 const { EXTERNAL_TRIGGER } = schema.config.regex;
 const idSchema = joi.reach(schema.models.job.base, 'id');
 
+/**
+ * Determine if this build is FIXED build or not.
+ * @method isFixedBuild
+ * @param  build
+ * @param  jobFactory
+ */
+async function isFixedBuild(build, jobFactory) {
+    const failureBuild = await jobFactory.get(build.jobId).then(job => job.getLatestBuild({ status: 'FAILURE' }));
+    const successBuild = await jobFactory.get(build.jobId).then(job => job.getLatestBuild({ status: 'SUCCESS' }));
```

Comment:
`jobFactory.get(build.jobId)` is called twice. It is wasteful. Why don't you add `const job = await jobFactory.get(build.jobId)` before these lines are executed?

Reply:
I think it's so wonderful.
Fixed.